app/assets/stylesheets/modules/flash.scss

```diff
@@ -0,0 +1,18 @@
+@import "variables";
+
+.flash-alert {
+  background-color: rgba( $red, .7)
```

Comment:
`.7` should be written with a leading zero as `0.7`<br>Expected 0 spaces between parentheses instead of 1<br>Declaration should be terminated by a semicolon